tests/unittests/GraphGradTest.cpp

```diff
@@ -237,3 +237,29 @@ TEST(GraphAutoGrad, checkMatMulToGradTest) {
   EE.compile(CompilationMode::Train, TF);
   EE.compile(CompilationMode::Infer, F);
 }
+
+/// Check that we can differentiate functions that use BatchedReduceAddNode.
+TEST(GraphAutoGrad, checkBatchedReduceAddGradTest) {
+  ExecutionEngine EE;
+  TrainingConfig TC;
+  PlaceholderBindings Bindings;
+
+  // Construct the network:
+  TC.learningRate = 0.001;
```

Comment:
It is not necessary to set this since the graph is never run, only compiled. You can delete it.